### ISSUE TITLE
Move Side by Side utilities into Layout section

### ISSUE THREAD

**Issue by chris-pearce** [westfieldlabs/scally#153]

**Comment by chris-pearce** [comment]
Doing this will 100% align with this: 

> > The rule is that all components sit inside layout modules which in most cases will be the grid.
> > https://github.com/westfieldlabs/scally/tree/master/components#how-to-use

I can't think of any other utils that called be classed as "layout"?

**Comment by kllevin** [comment]
I agree, these should be moved. However won't this be a breaking change?

**Comment by chris-pearce** [comment]
> > However won't this be a breaking change?

Yes.
